File: GitNexus/gitnexus/test/fixtures/lang-resolution/csharp-method-enrichment/Animal.cs
public abstract class Animal {
    public abstract string Speak();

    public static string Classify(string name) {
        return "mammal";
    }

    public bool Breathe() {
        return true;
    }
}

public class Dog : Animal {
    public override string Speak() {
        return "woof";
    }
}

File: GitNexus/gitnexus/test/fixtures/lang-resolution/kotlin-method-enrichment/Animal.kt
abstract class Animal {
    abstract fun speak(): String

    companion object {
        fun classify(name: String): String {
            return "mammal"
        }
    }

    fun breathe(): Boolean {
        return true
    }
}

class Dog : Animal() {
    override fun speak(): String {
        return "woof"
    }
}
